kornia/testing/__init__.py

```diff
@@ -279,7 +279,7 @@ def KORNIA_CHECK_SHAPE(x, shape: List[str]) -> None:
         start_idx: int = 1
```

Comment:
```suggestion
        start_idx = 1
```
define the variable before the if branch as `start_idx: int`